src/image_function_simd.cpp

```diff
@@ -941,6 +942,35 @@ namespace sse
         }
     }
 
+    void RgbToBgr( uint8_t * outY, const uint8_t * inY, const uint8_t * outYEnd, uint32_t rowSizeOut, uint32_t rowSizeIn, uint32_t RGB_simdSize, 
+                   const uint8_t colorCount, uint32_t simdWidth, uint32_t totalSimdWidth, uint32_t nonSimdWidth )
+    {
+        simd ctrl = _mm_set_epi8(15, 12, 13, 14, 9, 10, 11, 6, 7, 8, 3, 4, 5, 0, 1, 2);
+        for( ; outY != outYEnd; outY += rowSizeOut, inY += rowSizeIn ) {
+            const uint8_t * inX  = inY;
+            uint8_t       * outX = outY;
+
+            const uint8_t * outXEnd = outX + totalSimdWidth;
+
+            for( ; outX != outXEnd; outX += RGB_simdSize, inX += RGB_simdSize ) {
+                const simd * src = reinterpret_cast<const simd*> (inX);
+                simd * dst = reinterpret_cast<simd*> (outX);
+                simd result = _mm_loadu_si128( src );
+                result = _mm_shuffle_epi8(result, ctrl);
```

Comment:
[MacOS compilation](https://travis-ci.org/ihhub/penguinV/jobs/478124648) fails on this function. Check the bottom of the log.

Reply:
It's probably because this function is SSSE3.

Reply:
This function is declared in `tmmintrin.h` file which is not included here. Check [this](https://software.intel.com/en-us/node/524215)